tests/benchmark/app/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>dojo2-v0.2.0-keyed</title>
+	<link href="css/currentStyle.css" rel="stylesheet"/>
+</head>
+<body>
+	<div id="main"></div>
+	<script src="../../../../node_modules/@dojo/loader/loader.js"></script>
+	<script>
+		require.config({
+			packages: [
+				{ name: '@dojo', location: '../../../../node_modules/@dojo' },
+				{ name: 'pepjs', location: '../../../../node_modules/pepjs/dist', main: 'pep' }
+			]
+		});
+
+		require([ '../../../../_build/tests/benchmark/app/main' ], function () {});
```

Comment:
Is this path right? We should copy it over to _build as part of the grunt task?

Reply:
Fixed, is now using the built version